apps/rxjs/14-race-condition/src/app/app.component.ts

```diff
@@ -21,19 +23,22 @@ export class AppComponent implements OnInit {
   title = 'rxjs-race-condition';
   dialog = inject(MatDialog);
   topicService = inject(TopicService);
-  topics: TopicType[] = [];
+  topicSubject$ = new BehaviorSubject<TopicType[]>([]);
```

Comment:
This is one way of doing it, but you can do it in a declarative way. 

You can initiate topic directly inside your field initializer and avoid the ngOnInit hook